docs/core/compendium_de.md

```diff
@@ -0,0 +1,1177 @@
+    
+    Logo string
+    
+    URL string
+    
+    Image string
+    
+    ExposedPorts []ExposedPort
+    
+    ExposedCommands []ExposedCommand
+    
+    Volumes []Volume
+    
+    HealthCheck HealthCheck
+    
+    HealthChecks []HealthCheck
+    
+    ServiceAccounts []ServiceAccount
+    
+    Privileged bool
+    
+    Configuration []ConfigurationField
+    
+    Properties Properties
+    
+    EnvironmentVariables []EnvironmentVariable
+    
+    Dependencies []Dependency
+    
+    OptionalDependencies []Dependency
+}
+```
+
+### Name
+
+Name enthält den voll qualifizierten Namen des Dogus, der sich aus dem Dogu-Namensraum und dem einfachen Namen des Dogus
+zusammensetzt, begrenzt durch einen einfachen Schrägstrich "/". Dieses Feld ist obligatorisch.
+
+Der Dogu-Namespace ermöglicht es, den Zugriff auf Dogus in diesem Namespace zu regeln. Es gibt drei reservierte
+Dogu-Namespaces: Die Namespaces `official` und `k8s` sind für alle Benutzer ohne weitere Kosten zugänglich. Im Gegensatz
+dazu ist der Namespace `premium` nur für Abonnement-Nutzer zugänglich.
+
+Die Namespace-Syntax sollte aus folgenden Zeichen bestehen:
+
+- lateinische Kleinbuchstaben
+- Sonderzeichen Unterstrich "_", Minus "-"
+- Ziffern 0-9
+- einer Gesamtlänge von weniger als 200 Zeichen
+
+Der einfache Name von Dogu ermöglicht mehrere Adressierungsarten. Der einfache Name wird als Teil der URI des Cloudogu
+EcoSystems verwendet, um einen URI-Teil zu adressieren (wenn das Dogu eine exponierte UI bietet). Außerdem wird der
+einfache Name verwendet, um das Dogu nach dem Installationsprozess zu adressieren (z. B. um ein Dogu zu starten, zu
+stoppen oder zu entfernen), oder um generierte Ressourcen zu adressieren, die zum Dogu gehören.
+
+Die Syntax des einfachen Namens muss ein DNS-kompatibler Bezeichner sein und sollte aus folgenden Elementen bestehen:
+
+- lateinischen Kleinbuchstaben
+- Sonderzeichen Unterstrich "_", Minus "-"
+- Ziffern 0-9
+- eine Gesamtlänge von weniger als 20 Zeichen
+
+Es wird empfohlen, denselben voll qualifizierten Namen innerhalb der Dockerfiles des Dogus als Umgebungsvariable `NAME`
+zu verwenden.
+
+Beispiele:
+
+- official/redmine
+- premium/confluence
+- foo-1/bar-2
+
+### Version
+
+Version definiert die aktuelle Version des Dogus. Dieses Feld ist ein Pflichtfeld.
+
+Die Version folgt dem Format der semantischen Versionierung und ist zusätzlich in zwei Teile aufgeteilt. Die
+Anwendungsversion und die Dogu-Version.
+
+Ein Beispiel wäre 1.7.8-1 oder 2.2.0-4. Der erste Teil der Version (z. B. 1.7.8) steht für die Version der Anwendung (z.
+B. die nginx-Version in dem nginx-Dogu). Der zweite Teil steht für die Version des Dogus und sollte bei einer
+Erstveröffentlichung mit 1 beginnen (z. B. 1.7.8-1).
+
+Wenn sich die Anwendung nicht ändert, aber z.B. Änderungen im Startskript des Dogus vorgenommen werden, sollte die neue
+Version 1.7.8-2 sein. Wenn sich die Anwendung selbst ändert (z.B. ein nginx-Upgrade), sollte die neue Version 1.7.9-1
```

Comment:
Add Whitespace in between of abbreviations which contain respective whitespaces when unabbridged.
```suggestion
Version 1.7.8-2 sein. Wenn sich die Anwendung selbst ändert (z. B. ein nginx-Upgrade), sollte die neue Version 1.7.9-1
```